setup.py

```diff
@@ -51,7 +51,7 @@ def version(path):
     "IPython>=7.0.0",
     "jinja2",
     "nbformat",
-    "nbconvert",
+    "nbconvert<7.0.0",
```

Comment:
new version 7 was released on 16th. wheels for it are not available and it breaks during build. restricting it to under 7 til its stable.